packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -110,6 +123,8 @@ contract GuardiansRegistry is ControlledRecoverable, IGuardiansRegistry, ERC900,
     event GuardianDeactivationRequested(address indexed guardian, uint64 availableTermId, uint256 amount);
     event GuardianDeactivationProcessed(address indexed guardian, uint64 availableTermId, uint256 amount, uint64 processedTermId);
     event GuardianDeactivationUpdated(address indexed guardian, uint64 availableTermId, uint256 amount, uint64 updateTermId);
+    event GuardianDeactivationLockIncreased(address indexed guardian, address indexed lockManager, uint256 amount, uint256 total);
+    event GuardianDeactivationLockDecreased(address indexed guardian, address indexed lockManager, uint256 amount, uint256 total);
```

Comment:
Wonder if we can just use a single Changed event?

Reply:
Isn't going to increase the gas usage? cause we will need to encode one extra word parameterizing a boolean to tell whether the lock has increased or not, while this way simply increases the bytecode size. Although it's worth testing it